### PR TITLE
Added writing query and content to clipboard

### LINKED ISSUE #27: Feature Request: Copy current JSON displayed
This would be very useful, for example when entering `.[].timestamp` copy the JSON shown:

image

### PR DESCRIPTION
**Pull request by anton-suprun**
### Description

- Addresses issue: #27 
- Added two methods to Jnv
    - `content_to_clipboard` - writes currently visible json to clipboard and adds a hint that content was copied to clipboard
    - `query_to_clipboard` - writes current jq query to clipboard and adds a hint that the query was copied to clipboard

- Added two new shortcuts:
    - `ctrl-o` - copies content to clipboard
    - `ctrl-q` - copies query to clipboard 

### Screencasting

*Note: I have a clipboard manager (via Raycast); that's the pop-up you're seeing; I use it to show the latest copied text*

https://github.com/user-attachments/assets/9dfabb3b-530c-4956-979f-29f1d28c3eec

### Notes 

Please feel free to request any changes to code structure, shortcuts, etc. I am happy to receive any of your feedback on this 😊
Dependency promkit [PR](https://github.com/ynqa/promkit/pull/27)

Resolves #27

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -11,11 +11,12 @@ readme = "README.md"
 [dependencies]
 anyhow = "1.0.82"
 clap = { version = "4.5.4", features = ["derive"] }
+clipboard = "0.5.0"
 jaq-core = "1.2.1"
 jaq-interpret = "1.2.1"
 jaq-parse = "1.0.2"
 jaq-std = "1.2.1"
-promkit = "0.4.3"
+promkit = { path = "/Users/antonsuprun/Projects/promkit/promkit" }
```
Comment (anton-suprun) [inline Cargo.toml:19]: _note: will remove this once it's ready to merge_

**Comment by anton-suprun** [comment]
@ynqa 

Thanks again for your suggestion on the promkit change; let me know if you have any other suggestions :)

(Once again; apologies for the delay on this 🙏 )

**Comment by ynqa** [comment]
@anton-suprun LGTM! But please merge until promkit release (and edit the version after confirming release) 🙇

**Comment by anton-suprun** [comment]
> @anton-suprun LGTM! But please merge until promkit release (and edit the version after confirming release) 🙇

Thank you 😃 ! And yes, acknowledged. Will do that !

**Comment by ynqa** [comment]
@anton-suprun And also, please add the items about new key-bindings to https://github.com/ynqa/jnv/blob/main/README.md?plain=1#L96-L118.

**Comment by ynqa** [comment]
@anton-suprun I just released promkit as [v0.4.5](https://crates.io/crates/promkit/0.4.5) 🎉

**Comment by anton-suprun** [comment]
> @anton-suprun I just released promkit as [v0.4.5](https://crates.io/crates/promkit/0.4.5) 🎉 

Awesome!!
Will update this shortly 😁 (on mobile right now )

**Review by ynqa — APPROVED** [review]
LGTM! @anton-suprun Thanks for your contribution~ 👍

**Pull request merged**